README_py.md

```diff
@@ -0,0 +1,70 @@
+# Welcome to a Toy PyBind11 Algorithm
+
+## Why?
+The goal of this project is to allow users to take an algorithm written in C++ and translate that into runnable Python code.
+This could allow a user to develop C++ code and run it on the EmotiBit's firmware, software, and through Python hitting three different metrics.
+
+## Installation Steps
+* Follow [this link](https://github.com/EmotiBit/EmotiBit_Plugins) and clone the repository onto your machine.
+* Install Visual Studio 2022, this was tested successfully with 17.11.02.
+* Download ['Anaconda'](https://www.anaconda.com/download/) as shown here.
+* Open an ***Anaconda*** terminal by searching Anaconda Powershell Prompt on your machine and run the command below:
+```bash
+conda config --set channel_priority flexible
+```
+* Import the pyExample_alg01/EmotiBit-pyenv-modern.yml file into Anaconda Navigator by following Environments > Import.
+
+## Steps to Run Example
+* cd inside EmotiBit_Plugins/cppExample_alg01 and run the following commands ***inside an Anaconda terminal***:
+```bash
+conda activate EmotiBit-pyenv-modern
+mkdir build
+cd build
+cmake ..
+```
+* After cmake runs successfully, you will see a rounder.sln created in your build folder.
+  * After you open this file, navigate to the top of Visual Studio and find Build > Build Solution.
+  * This will create a rounder.pyd file in the release folder.
+* Now cd into the EmotiBit_Plugins/pyExample_alg01 folder and run the command below:
+```bash
+python3 pyExample_alg01.py <your roundable number here>
+```
+* The program should output a rounded version of your number!
+
+## How to Modify this to your Use Case
+* Looking at the code given you can see the following lines:
```

Comment:
This whole section just needs a complete rework. Lets discuss the goals and hopes/dreams of what this section wants to be in the next meeting.